lightning/src/ln/channel.rs

```diff
@@ -2906,13 +2918,13 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 		assert_eq!(self.channel_state & ChannelState::ShutdownComplete as u32, 0);
 
 		// BOLT 2 says we must only send a scriptpubkey of certain standard forms, which are up to
-		// 34 bytes in length, so don't let the remote peer feed us some super fee-heavy script.
-		if self.is_outbound() && msg.scriptpubkey.len() > 34 {
+		// 42 bytes in length, so don't let the remote peer feed us some super fee-heavy script.
```

Comment:
Maybe precise bip ref "A BIP-141-compliant witness program is at max 42 bytes in length, ..."